.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,50 @@
+---
+name: Bug report
+about: Did you find a bug in UMF? Please let us know.
+labels: "bug"
+---
+<!--
+Before creating new issue, ensure that similar issue wasn't already created
+  * Search: https://github.com/oneapi-src/unified-memory-framework/issues
+
+Note that if you do not provide enough information to reproduce the issue, we may not be able to take action on your report.
+Remember this is just a minimal template. You can extend it with data you think may be useful.
+-->
+
+<!-- fill the title of issue with short description -->
+
+## Environment Information
+
+- UMF version(s):                                             <!-- fill this out -->
```

Comment:
Perhaps a 'git tag' for now since we don't really have any versions released yet?